packages/web/app/src/index.css

```diff
@@ -36,6 +36,8 @@
     --ring: 215 20.2% 65.1%;
 
     --radius: 0.5rem;
+
+    --hive-content-height: calc(100vh - 84px - 47px);
```

Comment:
Let's have some comments describing these magic values or could we define this variable within the HTML so we don't have to use [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))?

Reply:
added in notion https://www.notion.so/theguildoss/Make-Laboratory-usable-d4d44ccb81b3417db7076d9f572ca12d

Reply:
added commit in my last PR https://github.com/kamilkisiela/graphql-hive/pull/5000 https://github.com/kamilkisiela/graphql-hive/pull/5000/commits/ebb25852547d27ab9c0a6f40ab1eb59251d39d73

also what do think about naming CSS variables 

- `--header-height`
- `--tabs-navbar-height`
- `--content-height`

Reply:
nice!